content/en/basics/the-blockchain/consensus/index.md

```diff
@@ -11,5 +11,49 @@ menu:
     identifier: "consensus-688d0e228f7debf695c906ad442cb462"
 weight: 100
 toc: true
+draft: false
 ---
 
+## Overview
+
+In the Filecoin blockchain, network _{{< tooltip "consensus" >}}_ is achieved using the Expected Consenus (EC) algorithm, a probabilistic, _{{< tooltip "Byzantine fault-tolerant" >}}_ consensus protocol. At a high-level, EC achieves consensus by running a secret, fair and verifiable _leader election_ at every {{< tooltip "epoch" >}} where a set number of participants may become eligible to submit a block to the chain based on fair and verifiable criteria. 
+
+## Properties of EC
+
+Expected Consensus (EC) has the following properties:
+
+- Each epoch has potentially multiple elected leaders who may propose a block.
+- A winner is elected randomly from a set of network participants weighted according to the respective storage power they contribute to the Filecoin network. 
+- All blocks proposed are grouped together in a _tipset_ (discussed below), from which the final chain is selected.
+- A block producer can be verified by any participant in the network.
+- The identity of a block producer is anonymous until they release their block to the network.
+
+## Steps in EC
```

Comment:
It's not necessary to callout _X of EC_ in each title. The reader knows we're talking about EC because we're on the _EC_ page.

Reply:
Removed in https://github.com/filecoin-project/filecoin-docs/pull/1831/commits/c0e4b87c18f60f33bfc86ff7194cb9c908690e80